### PR TITLE
Issue/3816 calc totals

### PR DESCRIPTION
**Pull request by martinchristov**
- [ ] Test plan | Unit test | Integration test
- [ ] Documentation

### CODE REVIEW

File: akvo/rsr/spa/app/modules/editor/section7/recipient-regions/recipient-regions.jsx [hunk 1]
```diff
@@ -114,6 +115,7 @@ const RecipientRegions = ({ formPush, validations, showRequired, errors }) => {
           </Button>
         )}
       />
+      <CalcTotalChecker section="7" path="recipientRegions" prop="percentage" />
```
Comment (punchagan) [inline akvo/rsr/spa/app/modules/editor/section7/recipient-regions/recipient-regions.jsx]: If there are recipient regions and countries both specified, the combined total should add up to 100%, not separately. 

See this [Zulip thread](https://akvo.zulipchat.com/#narrow/stream/131441-rsr/topic/Edit.20Error.20message.20IATI.20publication.20country.20percentage/near/173780667), for more information.

**Comment by punchagan** [comment]
image

May be we should move the recipient country and regions to be together? The remaining %age to be filled is not visible when even one of the the locations is expanded. 

Looks good, otherwise!

**Comment by martinchristov** [comment]
Ooh... I hadn't noticed this because the order I was looking at was:

1. Location items
2. Recipient countries
3. Recipient regions

However, there is a conditional in the code which puts Recipient countries on top for EUTF only.
Let's bring @AkvoAnt into this

**Comment by punchagan** [comment]
@martinchristov Do we still want to keep this open? Or just close this as stale?

**Pull request closed without merging**